backend/src/main/java/wooteco/prolog/login/aop/OnlyMember.java

```diff
@@ -0,0 +1,12 @@
+package wooteco.prolog.login.aop;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface OnlyMember {
```

Comment:
제가 영어를 잘 하는건 아니지만 이게 조금 더 자연스럽진 않나요?
```suggestion
public @interface MemberOnly {
```

Reply:
앗 ㅋㅋㅋㅋ 그러네요!! MemberOnly 가 훨씬 자연스럽군요!